spec/unit/provider/user/directoryservice_spec.rb

```diff
@@ -925,6 +925,10 @@ module Puppet::Util::Plist
       }
     end
 
```

Comment:
Should unit tests cover both branches of ` unless has_sha512_pbkdf2_authentication_authority_set?(users_plist)` ?